kratos/sources/accessor.cpp

```diff
@@ -0,0 +1,154 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//                   Riccardo Rossi
+//                   Carlos Roig
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/accessor.h"
+
+namespace Kratos
+{
+
+double Accessor::GetValue(
+    const Variable<double>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    return 0.0;
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+Vector Accessor::GetValue(
+    const Variable<Vector>& rVariable,
+    const Properties& rProperties,
+    const GeometryType& rGeometry,
+    const Vector& rShapeFunctionVector,
+    const ProcessInfo& rProcessInfo
+    ) const
+{
+    KRATOS_ERROR << "You are calling the GetValue of the virtual Accessor class..." << std::endl;
+    Vector aux(0);
+    return aux;
```

Comment:
This should also work since it is returning by copy, ryt?
```suggestion
    return Vector();
```